synapseclient/models/mixins/table_operator.py

```diff
@@ -2152,6 +2161,14 @@ async def upsert_rows_async(
                 is reached a `SynapseTimeoutError` will be raised.
                 The default is 600 seconds
 
+            wait_for_eventually_consistent_view: Only used if the table is a view. If
+                set to True this will wait for the view to be reflect any changes that
+                you've made to the view. This is useful if you need to query the view
+                after making changes to the data.
```

Comment:
```suggestion
            wait_for_eventually_consistent_view: Only used if the table is a view. If
                set to True this will wait for the view to reflect any changes that
                you've made to the view. This is useful if you need to query the view
                after making changes to the data.
```